backend/ethereum/channel/backend_test.go

```diff
@@ -113,40 +109,3 @@ func newChannelSetup(rng *rand.Rand) *test.Setup {
 		RandomAddress: createAddr,
 	}
 }
-
```

Comment:
Why you move this? This time unfortunately no explanation in commit message ;)

Reply:
Good catch  :smile: . I missed to add the explanation in the commit message

I removed this test because, it is replaced with `TestGenericMarshaler`.
Will update the commit message when squashing or rebasing.